## MethanoGram

Predicts culturing conditions of methanogens using [random forests](https://www.stat.berkeley.edu/~breiman/RandomForests/cc_home.htm) and [n-gram analysis](http://github.com/michbur/biogram). MethanoGram is trained using data from [PhyMet<sup>2</sup>](http://metanogen.biotech.uni.wroc.pl/), a database of the methanogenic bacteria.

**Usage**: upload 16S rRNA nucleotide sequences of unknown methanogens and press the "Submit" button. Sequences may be submitted directly or as .fasta files. Consult the *Mean error* panel to see results of the jackknife test.

For more information consult the [manual](http://phymet2.biotech.uni.wroc.pl/index.php?ind=help).

**Restrictions**:
  * Be patient - computations can take up to few minutes.  

**Author**: [Michal Burdukiewicz](http://www.smorfland.uni.wroc.pl/).

**Example 16S rRNA sequences**:
